tchap_red_list/__init__.py

```diff
@@ -11,36 +11,211 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from typing import Any, Dict
+import logging
+from typing import Any, Dict, Optional, Tuple
 
 import attr
-from synapse.module_api import ModuleApi
+from synapse.module_api import (
+    DatabasePool,
+    JsonDict,
+    LoggingTransaction,
+    ModuleApi,
+    UserProfile,
+    cached,
+    run_in_background,
+)
+
+logger = logging.getLogger(__name__)
+
+ACCOUNT_DATA_TYPE = "im.vector.hide_profile"
 
 
 @attr.s(auto_attribs=True, frozen=True)
 class RedListManagerConfig:
-    pass
+    discovery_room: Optional[str] = None
 
 
 class RedListManager:
```

Comment:
We could do with a docstring explaining how the module works, or maybe pointing back to the readme.

Reply:
I'm not sure there's much value in that, since I don't see a scenario where you're looking at this without having looked at the readme (and I'm not sure we do that in other modules).